### PR TITLE
Task#6  Feat : build content layout

### LINKED ISSUE #6: Create layout for Registration Form component
### Trello board 
[Task#6](https://trello.com/c/KEBhjsVR/2-task6-create-layout-for-registration-form-component)
### Layout
- [x] build content layout
- [x] Build interface design

### PR DESCRIPTION
**Pull request by TheHienDuong**
### Trello board 
[Task#6](https://trello.com/c/KEBhjsVR/2-task6-create-layout-for-registration-form-component)
### Element
- [x] `<form>`:
>- [x] `<label>`
>- [x] `<select>`
- [x] `<input>`
>- [x] 'type : radio'
>- [x] 'type : checked'
>- [x] 'type : textarea '
>- [x] 'type : submit'
### Demo
Screenshot 2023-05-08 102835

### CODE REVIEW

File: practice-01/index.html [hunk 1]
```diff
@@ -1,12 +1,109 @@
...
+
+    <!-- CSS -->
+    <link rel="stylesheet" href="./style/index.css" />
 </head>
+
 <body>
-<h1>hello</h1>
+    <main class="main">
+        <form method="POST" class="form">
+            <!-- fullName -->
```
Comment (phattruong-agilityio) [inline practice-01/index.html]: Uppercase

File: practice-01/index.html [hunk 2]
```diff
@@ -1,12 +1,109 @@
...
+                    <li class="form-check">
+                        <input name="answer" type="checkbox" /><span class="form-text">Angular</span>
+                    </li>
+                    <li class="form-check">
+                        <input name="answer" type="checkbox" /><span class="form-text">Spring</span>
+                    </li>
+                </ul>
+                <span class="form-message"></span>
+            </div>
+            <!-- description -->
```
Comment (phattruong-agilityio) [inline practice-01/index.html]: uppercase first character

File: practice-01/index.html [hunk 3]
```diff
@@ -1,12 +1,109 @@
...
+            <div class="form-group">
+                <label for="desc" class="form-label">Any comments or suggestions (Max 500 characters)</label>
+                <textarea name="desc" id="desc" rows="5"></textarea>
+                <span class="form-message"></span>
+            </div>
+            <!-- Button -->
+            <button class="form-submit">Submit</button>
```
Comment (phattruong-agilityio) [inline practice-01/index.html]: button type submit

**Comment by phattruong-agilityio** [comment]
Did you have any question while working with this?

**Comment by TheHienDuong** [comment]
> Did you have any question while working with this?

I have a question about this.
I don't see a validator request for yes/no question and checkbox form. So what do I need to do with them.

File: practice-01/index.html [hunk 4]
```diff
@@ -1,12 +1,109 @@
...
+                </ul>
+                <span class="form-message"></span>
+            </div>
+            <!-- Checkbox -->
+            <div class="form-group">
+                <label id="question-checkbox" class="form-label">Languages and Frameworks known (Check all that
+                    apply)</label>
+                <ul class="form-list">
+                    <li class="form-check">
+                        <input name="answer" type="checkbox" /><span class="form-text">C</span>
```
Comment (phattruong-agilityio) [inline practice-01/index.html]: duplicate name="answer"

**Pull request merged**